pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/LiteralsFirstInComparisonsRule.java

```diff
@@ -25,40 +23,37 @@ public class LiteralsFirstInComparisonsRule extends AbstractJavaRulechainRule {
               "compareTo",
               "compareToIgnoreCase",
               "contentEquals");
+    private static final String EQUALS = "equals";
 
     public LiteralsFirstInComparisonsRule() {
         super(ASTMethodCall.class);
     }
 
     @Override
     public Object visit(ASTMethodCall call, Object data) {
-        if ("equals".equals(call.getMethodName())
-            && call.getArguments().size() == 1
```

Comment:
double trouble `call.getArguments().size() == 1`

<img width="609" alt="image" src="https://github.com/user-attachments/assets/4f4e82ce-8a7e-48c4-a733-abc5a007fbaf" />

Reply:
@oowekyala is this any good?